README.md

```diff
@@ -24,19 +24,16 @@ The action is intended to be run on a cron schedule, see [mirror.yml](./.github/
 ```
 on:
   schedule:
-    cron:
-      # cron actions will not run more frequently than once every 5 minutes
-      - '*/5 * * * *'
+    - cron: '*/5 * * * *' # 5 minutes is the smallest frequency available to actions
 jobs:
   mirror_column:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - uses: jonabc/linked-project-columns@<version>
+    - uses: jonabc/linked-project-columns@v1.0.0
```

Comment:
If this is going to promote a specific version, I'd prefer to promote a major tag so people get non-breaking changes automatically

```suggestion
    - uses: jonabc/linked-project-columns@v1
```